mod-kb-ebsco/mod-kb-ebsco-java.postman_collection.json

```diff
@@ -118,12 +118,17 @@
 									"   }else{",
 									"       reject(err, echoGetRequest);",
 									"   }",
-									"      }, 5000);",
+									"      }, 7000);",
```

Comment:
Just curious as to why this timeout needed to be increased? Wonder if environment variable may be a good place for holding this if this needs to be tweaked frequently. Although this is the only place it is referenced

Reply:
Tests pass twice. One time via Postman, second time via Newman.This timeout needed for uploading all schemes for tests. With Postman it is okay. Newman does not upload all schemes for that time. As solution I have increased timeout for uploading schemes.

Reply:
Ok thanks @stanislav-mureiko for the info